generators/app/templates/custom/_login.html

```diff
@@ -1,36 +1,38 @@
 <ion-modal-view>
-  <ion-header-bar class="bar bar-header bar-positive">
-    <h1 class="title" translate="login.title">Sign In</h1>
-    <button class="button button-clear button-primary" ng-click="vm.hideModal()" translate="entity.action.cancel">Cancel</button>
-  </ion-header-bar>
-  <ion-content class="padding">
-    <div class="list">
-      <label class="item item-input">
-        <span class="input-label" translate="entity.action.cancel">Login</span>
-        <input type="text" class="form-control" id="username" placeholder="{{'global.form.username.placeholder' | translate}}"
```

Comment:
We need the space before the `| translate` so that the translate regex can remove it for apps without i18n.  Also please add the space back in the password's placeholder.